app/controllers/products_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProductsController < ApplicationController
+  def index
+    @search = Product.all.ransack params[:q]
+    @products = Product.all.page(params[:page]).per(params[:limit])
```

Comment:
sao ko lấy lại data search ở trên sẽ tốt hơn chứ e, nếu có filter thì xử lý ko thì trả về all
```suggestion
    @products = @search.page(params[:page]).per(params[:limit])
```